### PR TITLE
Adding 'Implementing connectivity discovery script'

### PR DESCRIPTION
**Pull request by jaklke**
Page added to existing section Advanced configuration for Administrators, but please rearrange as discussed via mail.

### CODE REVIEW

**Comment by MariekeGO** [comment]
@jaklke Could you check if the reviewed version is OK to merge? Note that I moved the info about elements being a digital twin of the equipment to the "About elements" page and added a link on the new page instead.

File: user-guide/Standard_Apps/IDP/Advanced_configuration/Implementing the Connectivitivty Discovery script.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+If the equipment runs such a network protocol, the connector will have or will need to be extended with the information above. The custom script can then provide IDP with the following information regarding neighbors of the equipment:
+
+- The DataMiner and element ID of the element.
+- The unique identifier (i.e. the Local Chassis ID) of the equipment.
+- For each neighbor of the equipment:
+
+  - The DCF interface ID on which the neighbor is detected.
+  - The local port ID on which the neighbor is connected.
+  - The remote chassis ID of the neighbor.
+
```
Comment (jaklke) [inline user-guide/Standard_Apps/IDP/Advanced_configuration/Implementing the Connectivitivty Discovery script.md:50]: - the remote port ID (this is port ID which uniquely identifies the interface on the neighbor)

File: user-guide/Standard_Apps/IDP/Advanced_configuration/Implementing the Connectivitivty Discovery script.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- The destination DMA and element ID
+- The destination DCF interface ID
+
+At the same time, the custom script is linked to the CI Type and should only use the information available in the element itself.
+
+A network protocol that allows the equipment to discover neighboring network equipment (like LLDP, CDP, etc.) will have:
+
+- A unique way to identify equipment. The equipment will be given an identifier that no other equipment is using. We refer to this as the **Local Chassis ID**.
+- A unique way to identify interfaces on the equipment. This can be different from the typical interface ID. We refer to this as a **Local Port ID**.
+- The ability to see on which interfaces neighbors are detected. The neighbors are detected with their unique identifiers. From the perspective of local equipment, these are **Remote Chassis IDs**.
```
Comment (jaklke) [inline user-guide/Standard_Apps/IDP/Advanced_configuration/Implementing the Connectivitivty Discovery script.md:39]: The local port IDs of the neighbor, are perceived as Remote Port IDs.

**Review by jaklke — COMMENTED** [review]
Please see comments

**Comment by MariekeGO** [comment]
> Please see comments

Your comments have been implemented. OK to merge?

**Comment by jaklke** [comment]
OK to merge. Thanks.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**